controllers/openstackdataplanenodeset_controller.go

```diff
@@ -304,6 +320,7 @@ func (r *OpenStackDataPlaneNodeSetReconciler) Reconcile(ctx context.Context, req
 		logger.Info("Set NodeSet DeploymentReadyCondition true")
 		instance.Status.Conditions.MarkTrue(condition.DeploymentReadyCondition,
 			condition.DeploymentReadyMessage)
+		instance.Status.DeployedConfigHash = configHash
```

Comment:
I have deployed nodeset and ran this command to patch nodeset:
oc patch osdpns/openstack-edpm -p '{"spec": {"nodeTemplate": {"ansible": {"ansibleVars": { "dns_search_domains": ["bne-home.com"] }}}}}' --type=merge

Is it desired that DeployedConfigHash is equal to configHash right after patch is applied. There was no openstackdeployemnt started or anything.

Reply:
So these two values should only be different if you already have a `OpenStackDataPlaneDeployment` that has completed, and then the config is changed.

If there is no deployment at all though, I guess we shouldn't have any value in the DeployedConfigHash at all. Not until a deployment is created and finished. 

I'll double check the code there.

Reply:
Looks like this part I'm describing works fine. I think the issue might be with the length of the hash that's being generated. I'll see if I can shorten that and hope that solves the issues I'm seeing with functional tests.